src/components/link.js

```diff
@@ -0,0 +1,39 @@
+var registerComponent = require('../core/component').registerComponent;
+
+module.exports.Component = registerComponent('link', {
+  schema: {
+    url: { default: '' },
```

Comment:
Maybe `href` to be consistent with the primitive that is abstracting it as with as with `<a>`?